.pylintrc

```diff
@@ -60,13 +60,11 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
```

Comment:
@Luca409: can we make tabbing consistent.  I think we prefer to keep the warnings globally but disable specific lines where we think it's unavoidable (and these two cases certainly show up a few times).

Reply:
Sorry, I'm not sure what you mean by this.